### PR TITLE
feat: deployer user in the main account

### PR DESCRIPTION
**Pull request by jkrnak**

### CODE REVIEW

File: core-aws-main/iam.tf [hunk 1]
```diff
@@ -33,3 +33,26 @@ module "iam_assume_groups" {
...
+data "aws_iam_policy_document" "ci_deployer_assume_policies" {
+  statement {
+    actions = ["sts:AssumeRole"]
+    #tfsec:ignore:aws-iam-no-policy-wildcards => The default wildcard has some filtering in, doesn't allow all roles
+    resources = var.ci_deployer_role_arns
+  }
+}
+
+resource "aws_iam_user_policy" "ci_deployer_assume" {
+  name   = "assume-policies"
```
Comment (bpesics) [inline core-aws-main/iam.tf]:
```suggestion
  name   = "iam-assume-role_deployers"
```

**Review by bpesics — APPROVED** [review]
apart from the policy name 👍

File: core-aws-main/iam.tf [hunk 2]
```diff
@@ -33,3 +33,25 @@ module "iam_assume_groups" {
...
+}
+
+resource "aws_iam_access_key" "ci_deployer" {
+  user = aws_iam_user.ci_deployer.name
+}
+
+data "aws_iam_policy_document" "ci_deployer_assume_policies" {
+  statement {
+    actions   = ["sts:AssumeRole"]
+    resources = var.ci_deployer_role_arns
```
[the remark this answers is not in the document]
Comment (bpesics) [reply core-aws-main/iam.tf:50]: I think the `ignore` needs to be on the previous line for tfsec

[the remark this answers is not in the document]
Comment (jkrnak) [reply core-aws-main/iam.tf:50]: it worked where it is, no errors anymore

[the remark this answers is not in the document]
Comment (jkrnak) [reply core-aws-main/iam.tf:50]: it is on the previous line

**Pull request merged**